2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -86,7 +91,49 @@ id: \`${id}\`
     // [END v2SendToDiscord]
   } catch (error) {
     logger.error(
-        `Unable to post fatal Crashlytics alert ${id} to Discord`,
+        `Unable to post fatal Crashlytics alert ${id} for ${appId} to Discord`,
+        error,
+    );
+  }
+});
+
+/**
+ * function triggered by AppDistribution that publishes a message
```

Comment:
Add space between "App" and "Distribution"

Reply:
Done.